apps/showcase/src/app/sdk/sdk.template.html

```diff
@@ -0,0 +1,50 @@
+<h1>SDK Generator</h1>
+<div class="row">
+  <div class="right-nav order-1 order-lg-2 col-12 col-lg-2 sticky-lg-top pt-5 pt-lg-0">
+    <o3r-in-page-nav-pres
+      id="sdk-nav"
+      [links]="links$ | async"
+    >
+    </o3r-in-page-nav-pres>
+  </div>
+  <div class="order-2 order-lg-1 col-12 col-lg-10">
+    <div class="d-flex flex-column gap-2 align-items-center justify-center flex-fill alert alert-warning">
+      <img src="assets/otter-in-progress.svg" width="128" height="128" aria-hidden="true">
+      <p class="h2">Work in progress</p>
```

Comment:
Work is still in progress when PR will be merged ?

Reply:
removed